src/main/scala/is/hail/HailContext.scala

```diff
@@ -312,21 +312,23 @@ class HailContext private(val sc: SparkContext,
     types: java.util.HashMap[String, Type],
     commentChar: String,
     separator: String,
+    quote: java.lang.Character,
```

Comment:
If the Java side is char, py4j passes the first character of the python side.  If the string is empty on the python side, py4j throws `StringIndexOutOfBoundsException`.